src/gui/Gui.cc

```diff
@@ -148,11 +148,21 @@ std::unique_ptr<ignition::gui::Application> createGui(
     return nullptr;
 
   std::size_t runnerCount = 0;
+  std::string defaultGuiConfigName = "gui.config";
 
   // Configuration file from command line
   if (_guiConfig != nullptr && std::strlen(_guiConfig) > 0)
   {
-    if (!app->LoadConfig(_guiConfig))
+    // The playback flag (and not the gui-config flag) was
+    // specified from the command line
+    if (std::string(_guiConfig) == "_playback_")
+    {
+      ignition::common::env(IGN_HOMEDIR, defaultConfig);
+      defaultConfig = ignition::common::joinPaths(defaultConfig, ".ignition",
```

Comment:
I think you need to move this logic up, before the App calls `SetDefaultConfigPath`

Reply:
39ca2b20e1f33e08d173513b83b827e692b73f82